access/acceptance/permissions_test.go

```diff
@@ -91,3 +92,78 @@ func TestAccDatabricksPermissionsResourceFullLifecycle(t *testing.T) {
 		},
 	})
 }
+
+func TestAccDatabricksJobPermissionsResourceFullLifecycle(t *testing.T) {
+	randomName := acctest.RandStringFromCharSet(10, acctest.CharSetAlphaNum)
+	acceptance.AccTest(t, resource.TestCase{
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+				resource "databricks_job" "this" {
+					name = "First %[1]s"
+					timeout_seconds = 3600
+					max_retries = 1
+					max_concurrent_runs = 1
+
+					new_cluster  {
+						num_workers   = 1
+						spark_version = "6.6.x-scala2.11"
+						node_type_id  = "Standard_DS3_v2"
+					}
+				}
+
+				resource "databricks_group" "first" {
+					display_name = "First %[1]s"
+				}
+
+				resource "databricks_permissions" "dummy" {
+					job_id = databricks_job.this.id
+					access_control {
+						group_name = databricks_group.first.display_name
+						permission_level = "CAN_MANAGE"
+					}
+				}`, randomName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("databricks_permissions.dummy",
+						"object_type", "job"),
+					acceptance.ResourceCheck("databricks_permissions.dummy",
+						func(client *common.DatabricksClient, id string) error {
+							job, err := compute.NewJobsAPI(client).Read(id)
+
+							if err != nil {
+								return err
+							}
+							userName := job.CreatorUserName
+
+							userACL := AccessControlChange{
+								UserName:        &userName,
+								PermissionLevel: "IS_OWNER",
+							}
+
+							accessControlChange := []*AccessControlChange{&userACL}
+
+							jobACL := AccessControlChangeList{
+								accessControlChange,
+							}
+
+							param := &jobACL
+							NewPermissionsAPI(client).SetOrDelete(fmt.Sprintf("/jobs/%s/", id), param)
+
+							permissions, err := NewPermissionsAPI(client).Read(id)
+							if err != nil {
+								return err
+							}
+							for _, acl := range permissions.AccessControlList {
+								for _, permission := range acl.AllPermissions {
+									println(permission.PermissionLevel)
```

Comment:
replace `println` with `t.Printf`, so that output is not polluted

Reply:
on the other hand, you can `t.Printf("%s", permissions)`, which will in fact call `permissions.String()` and display permissions

Reply:
You mean fmt.Printf ?